packages/tests/src.ts/test-contract.spec.ts

```diff
@@ -340,6 +340,66 @@ describe("Test Contract Transaction Population", function() {
         });
         assert.strictEqual(BigNumber.from(balance).toNumber(), 10000, 'balance mismatch');
     }).timeout(60000);
+
+   it('should have a .wait function', async function() {
```

Comment:
We should assert the exact log being emitted not only that it is populated (!= null)

Reply:
I made the test more explicit